### ISSUE TITLE
sympy can't solve differential equation

### ISSUE THREAD

**Issue by awa5114** [sympy/sympy#16515]
I am relatively new to sympy but well versed in python. Please take a look at the following, this to me seems like a simple pde but sympy cannot solve it

```
from sympy import *
from sympy.abc import r, t

init_printing()
u, R, T = map(Function, 'uRT')
c_h = symbols('c_h')

eq = Eq((1/r)*u(r,t).diff(r)+u(r,t).diff(r,2)-u(r,t).diff(t))

pdsolve(eq)
```

I get an `NotImplementedError` related to the fact that no `hint` can be found since

`classify_pde(eq)` returns an empty tuple.

Am I doing something wrong or is this indeed a bug?  Thanks.

**Comment by jksuom** [comment]
> Am I doing something wrong or is this indeed a bug?

It looks like currently solvers for only first order equations have been implemented: https://docs.sympy.org/latest/modules/solvers/pde.html, last paragraph.

**Comment by awa5114** [comment]
I see. Are higher order pde's on the feature list? It seems like an important functionality, or is it the plan to focus on other priorities? If so what are they? Thanks.

**Comment by oscarbenjamin** [comment]
There is a lot of work going on in SymPy right now on all sorts of things (look at the issues/PRs). There isn't really a plan of time/work allocation since everything is contributed by volunteers. If you would like to improve the pde module then feel free to send a PR.

**Comment by awa5114** [comment]
I might be tempted to but I'm not proficient enough yet to provide any significant contribution. I was hoping to master the package first. Where is most of the work currently headed in sympy? Is the goal to make it handle ever more complex algebra/calculus/symbolic analytical mathematics or are we headed somewhere else?

**Comment by oscarbenjamin** [comment]
Yes handling more complicated things is a goal. Here's a roadmap:
[https://www.sympy.org/zh/roadmap.html](https://www.sympy.org/zh/roadmap.html)

Some things are out of scope but otherwise any particular thing is just done if someone is willing to do it.

**Comment by awa5114** [comment]
interesting. I will take a look once I become more proficient with the package.
